### PR TITLE
{bio}[dummy/dummy] picard v2.6.0, picard v2.10.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/4bd02a459e96c373b5b800af2d652696 for a full test report.

File: easybuild/easyconfigs/p/picard/picard-2.10.0-Java-1.8.0_131.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://sourceforge.net/projects/picard'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = ['%(name)s.jar']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/p/picard/picard-2.10.0-Java-1.8.0_131.eb]: Checksum still missing

File: easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = ['%(name)s.jar']
+
+dependencies = [('Java', '1.8.0_131')]
+
+checksums = [('sha256', 'e256d5e43656b7d8be454201a7056dce543fe9cbeb30329a0d8c22d28e655775')]
```
Comment (vanzod) [inline easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb]: Move after `sources`

File: easybuild/easyconfigs/p/picard/picard-2.6.0-Java-1.8.0_131.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = ['%(name)s.jar']
+
+dependencies = [('Java', '1.8.0_131')]
+
+checksums = [('sha256', '671d9e86e6bf0c28ee007aea55d07e2456ae3a57016491b50aab0fd2fd0e493b')]
```
Comment (vanzod) [inline easybuild/easyconfigs/p/picard/picard-2.6.0-Java-1.8.0_131.eb]: Move after `sources`

File: easybuild/easyconfigs/p/picard/picard-2.10.0-Java-1.8.0_131.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/p/picard/picard-2.10.0-Java-1.8.0_131.eb]: @vanzod yes, that's deliberate. I'd like to remove this one from the PR, but not sure what's the cleanest way of doing that...

Comment (vanzod) [inline easybuild/easyconfigs/p/picard/picard-2.10.0-Java-1.8.0_131.eb]: Just delete the file in your local branch, commit the deletion and push.

File: easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+version = '2.10.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://sourceforge.net/projects/picard'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = ['%(name)s.jar']
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb]: Use the rename-after-download support to properly version this?

You'll also need to include `postinstallcmds` to rename the install file to `picard.jar`?

File: easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+checksums = [('sha256', 'e256d5e43656b7d8be454201a7056dce543fe9cbeb30329a0d8c22d28e655775')]
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb]: drop the `'sha256'`, it's a default checksum format:

```python
checksums = ['e256d5e43656b7d8be454201a7056dce543fe9cbeb30329a0d8c22d28e655775']
```

File: easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb (hunk 4 above)
Comment (verdurin) [inline easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb]: @boegel I very nearly added that support, until I thought it through and couldn't immediately think of how to rename the file back again. Forgot about `postinstallcmds` - will use that.

**Comment by verdurin** [comment]
Is that okay now @boegel ? Looks much cleaner to me.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2570.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9317e42d0e298642a6041cfc2d339b22 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**